task/buildah-remote/0.1/buildah-remote.yaml

```diff
@@ -184,7 +201,9 @@ spec:
         BUILDAH_ARGS="--pull=never"
         UNSHARE_ARGS="--net"
         for image in $(grep -i '^\s*FROM' "$dockerfile_path" | sed 's/--platform=\S*//' | awk '{print $2}'); do
-          unshare -Ufp --keep-caps -r --map-users 1,1,65536 --map-groups 1,1,65536 -- buildah pull $image
+          if [ "${image}" != "scratch" ]; then
```

Comment:
This seems like an unrelated change? Same in `task/buildah/0.1/buildah.yaml`

Reply:
This is https://issues.redhat.com/browse/RHTAPBUGS-1066, maybe it should be moved to its own PR.

Reply:
Pulled into https://github.com/redhat-appstudio/build-definitions/pull/723